### PR TITLE
Resolve MUWM-4489 - fix a bug in commit 3ea16cad8

### PR DESCRIPTION
**Pull request by fanglinfang**

### CODE REVIEW

File: myuw/dao/registration.py [hunk 1]
```diff
@@ -2,6 +2,7 @@
 This module provides access to uw_sws registration module
 """
 
+from copy import deepcopy
```
Comment (ezturner) [inline myuw/dao/registration.py]: Why is deepcopy necessary here?

Comment (ezturner) [reply myuw/dao/registration.py]: Oh, because you're altering a list you're iterating through?

Comment (fanglinfang) [reply myuw/dao/registration.py]: Because removing the list item will mess up the list (some item may be skipped)

Comment (ezturner) [reply myuw/dao/registration.py]: I think you might want copy() there instead, since you aren't modifying the instructor objects, merely the list

**Pull request merged**